dashboard/src/components/users/UsersTable.vue

```diff
@@ -43,11 +44,11 @@ const userInfo = useUserInfo()
 const exportUtil = useExportUtil()
 const numberFormat = useNumberFormat()
 const projConfig = useProjConfig()
+const projectDetailsState = useProjDetailsState()
 
 let filters = ref({
   user: '',
-  progress: 0,
-  minimumPoints: 0
+  progress: [0, 100],
```

Comment:
This looks nice, but could sometimes be unreadable. A obj with range method names will make the code more intuitive:

```js
progressRange: {
 min: 0,
 max: 100,
}
```